services/web/server/src/simcore_service_webserver/garbage_collector_core.py

```diff
@@ -194,6 +194,7 @@ async def remove_disconnected_user_resources(
                                 "first_name": "garbage",
                                 "last_name": "collector",
                             },
+                            suppress_project_locked_error=True,
```

Comment:
this guy already has a try...catch therefore just add one. or otherwise add a force argument instead.